public_html/dashboard.js

```diff
@@ -0,0 +1,79 @@
+// registers all event listeners and logic for the user portal dashboard
+$(document).ready(function () {
+  // initialize the dashboard with none of the tabs showing
+  $('#contactsTab').css ('display', 'none');
```

Comment:
Thats a good question, that was solving the problem of hiding elements on start up but this may be able to be changed to .hide() now that we aren't re rendering everything anymore.  I will look into it.